gravity-forms/gw-custom-activation-template.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+* Gravity Forms Custom Activation Template
+* http://gravitywiz.com/customizing-gravity-forms-user-registration-activation-page
+*/
+add_action( 'wp', 'custom_maybe_activate_user', 9 );
+function custom_maybe_activate_user() {
+
+	$template_path    = STYLESHEETPATH . '/gfur-activate-template/activate.php';
+	$is_activate_page = isset( $_GET['page'] ) && $_GET['page'] === 'gf_activation';
+	$is_activate_page = $is_activate_page || isset( $_GET['gfur_activation'] ); // WP 5.5 Compatibility
```

Comment:
There is a potential for a notice here if the the page parameter is not set on line 11. Thoughts on combining these two lines like so?

```
$is_activate_page = isset( $_GET['page'] ) && in_array( $_GET['page'], array( 'gf_activation', 'gfur_activation' ), true );
```

Reply:
@spivurno I thought the change was doing away with the `page` parameter in favor of a new `gfur_activation` parameter, is this incorrect?

If page parameter is empty or not set to `gf_activation`, line 11 would just check for the existence of the `gfur_activation` parameter and set `is_activate_page` to `true`, in all other cases it should be `false` and the custom template wouldn't be loaded.

Reply:
Dude, you're 💯 % right. Please ignore me. 😆